utils/build-presets.ini

```diff
@@ -1804,7 +1804,7 @@ enable-tensorflow
 
 [preset: tensorflow_linux,no_test]
 mixin-preset=buildbot_linux,no_test
-enable-tensorflow
+enable-tensorflow-gpu
```

Comment:
Is this change intentional?

Reply:
Oops, thanks for the catch. Fixed in https://github.com/apple/swift/pull/17839/commits/9d8a1312e2e3027e17a12bf2d796a29e4f05ebb0.